libraries/core-react/src/components/Snackbar/Snackbar.tokens.ts

```diff
@@ -17,46 +17,48 @@ const {
     },
   },
   spacings: {
-    comfortable: { medium: spacingMedium },
+    comfortable: { medium: spacingMedium, x_large: spacingXLarge },
   },
   elevation: { overlay: boxShadow },
   clickbounds: { default__base: clickbounds },
+  shape: {
+    button: { borderRadius: radius },
+  },
 } = tokens
 
-type Snackbar = {
-  background: string
+type Snackbar = ComponentToken & {
   boxShadow: string
-  minHeight: string
-  spacings: {
-    left: string
-    bottom: string
-    padding: string
-    actionSpace: string
-  }
-  text: {
-    color: string
-    typography: Typography
-  }
-  borderRadius: string
-  buttonColor: string
 }
 
 export const snackbar: Snackbar = {
   background,
   boxShadow,
   minHeight: clickbounds,
+  border: {
+    type: 'border',
+    width: 0,
+    radius,
+  },
+  entities: {
+    actions: {
+      spacings: {
+        left: spacingXLarge,
+        top: '10px',
+        bottom: '10px',
```

Comment:
These spacings values look to be wrong? They are also abit superflous as we are setting `minHeight`.

Reply:
These are negative top/bottom values in `SnackbarAction` - spacing buttons to `Snackbar` is `6px`, but since padding already is `16px` without buttons, we need `-10px` on the buttons 🤔 And apparently negative values wasn't valid in the token